src/tests/functional/inference_engine/transformations/common_optimizations/pull_through_reduce_test.cpp

```diff
@@ -0,0 +1,286 @@
+// Copyright (C) 2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <gtest/gtest.h>
+#include <string>
+#include <memory>
+
+#include <openvino/core/model.hpp>
+#include <openvino/opsets/opset9.hpp>
+#include <openvino/pass/manager.hpp>
+#include <transformations/common_optimizations/pull_through_reduce.hpp>
+#include <transformations/init_node_info.hpp>
+#include <transformations/utils/utils.hpp>
+
+#include "common_test_utils/ngraph_test_utils.hpp"
+
+using namespace testing;
+using namespace ov;
+using namespace opset9;
+
+namespace {
+    template<typename ReduceType>
+    std::shared_ptr<Model> gen_unsqueeze_model(element::Type in_type,
```

Comment:
Avoid shortcuts if doable, 'gen' typically comes from nouns, while in here from verb.